src/treeview/node.js

```diff
@@ -106,6 +106,26 @@ export default class Node {
 	}
 
 	/**
+	 * Returns ancestors array of this node.
+	 *
+	 * @param {Boolean} [includeNode=false] When set to `true` this node will be also included in parent's array.
+	 * @param {Boolean} [parentFirst=false] When set to `true`, array will be sorted from node's parent to root element,
+	 * otherwise root element will be the first item in the array.
+	 * @returns {Array} Array with ancestors.
+	 */
+	getAncestors( includeNode, parentFirst ) {
```

Comment:
Maybe `options` objects would be better? `getAncestors( { parentFirst: true } )` seems to be more readable them `getAncestors( false, true )`. And more scaleable.